packages/api-plugin-fulfillment-method-pickup-store/README.md

```diff
@@ -0,0 +1,70 @@
+# api-plugin-fulfillment-method-pickup-store
+
+[![npm (scoped)](https://img.shields.io/npm/v/@reactioncommerce/api-plugin-fulfillment-method-pickup-store.svg)](https://www.npmjs.com/package/@reactioncommerce/api-plugin-fulfillment-method-pickup-store)
+[![CircleCI](https://circleci.com/gh/reactioncommerce/api-plugin-fulfillment-method-pickup-store.svg?style=svg)](https://circleci.com/gh/reactioncommerce/api-plugin-fulfillment-method-pickup-store)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+
+## Summary
+
+This plugin Implements Store pickup as a fulfillment method under the type pickup.
+
+## Included in this example plugin
+ß
+### `src/`
+
+The `src` folder contains all the plugin files. An `index.js` with a bear-bones `registerPlugin` is included.
```

Comment:
This is the template from the example plugin, please update with relevant information about this plugin

Reply:
Done